### PR TITLE
Add a few logo pictures for later use

### PR DESCRIPTION
**Pull request by Chasen-Zhang**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary
Reserve a few logo pictures for later use
## Changelog

- Other 

## Test Plan

Unit Tests

Stateless Tests

### CODE REVIEW

**Comment by Xuanwo** [comment]
Hi, is it possible to add a more understandable suffix for files?

like `xxx.png` to `xxx-64x64.png` or `xxx-small.png` or `xxx-black.png`?

**Comment by Chasen-Zhang** [comment]
> Hi, is it possible to add a more understandable suffix for files?
> 
> like `xxx-1.png` to `xxx-64x64.png` or `xxx-small.png` or `xxx-black.png`?

I didn't notice that. Thanks for reminding me

**Comment by Chasen-Zhang** [comment]
I plan to use the position of the text on the logo, horizontal or vertical

File: website/static/img/logo/databend-logo-horizontal.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...2,8.025,6.383,21,3.135C22.194,19.843,42.528,14.691,50.225,19.3Z" style="fill:#52adff"/><path d="M39.932,12.068c.977.45,2.681.8,3.6,1.352-7.366-2.662-15.256-2.372-21.316-1.512A17.8,17.8,0,0,1,12.2,10.545a39.493,39.493,0,0,1,22.923-.279A35.117,35.117,0,0,1,39.932,12.068Z" style="fill:#015bce"/><path d="M43.436,13.359A24.2,24.2,0,0,1,50.225,19.3c-7.7-4.6-28.032.547-28.032.547-12.972,3.249-19.634-1.9-21-3.135a33.483,33.483,0,0,1,11.654-6.38,17.805,17.805,0,0,0,10.02,1.362A42.2,42.2,0,0,1,43.436,13.359Z" style="fill:#0175f5"/><path d="M40.58,11.851a29.994,29.994,0,0,1,2.854,1.508,42.191,42.191,0,0,0-20.565-1.668,17.805,17.805,0,0,1-10.02-1.362,39.493,39.493,0,0,1,22.923-.279A35.014,35.014,0,0,1,40.58,11.851Z" style="fill:#015bce"/><path d="M22.257,1.463a1.474,1.474,0,0,1-.9,1.351v0A6.252,6.252,0,0,0,18.524,5.65,6.258,6.258,0,0,1,19.544.7c.053-.078.109-.149.168-.223A1.443,1.443,0,0,1,20.787,0a1.464,1.464,0,0,1,1.47,1.459Z" style="fill:#015bce"/><path d="M17.2,7.135c0,.23-.015.457-.037.681a6.137,6.137,0,0,0-2.576-1.571.24.24,0,0,0-.071-.019A1.557,1.557,0,1,1,16.346,4,6.175,6.175,0,0,1,17.2,7.135Z" style="fill:#015bce"/><circle cx="23.318" cy="27.389" r="1.337" style="fill:#0175f5"/></svg>
```
Comment (BohuTANG) [inline website/static/img/logo/databend-logo-horizontal.svg:1]: Please rename it in English.

Comment (Chasen-Zhang) [reply website/static/img/logo/databend-logo-horizontal.svg:1]: 👌🏻   I'll sync up with the designer

**Review by Xuanwo — APPROVED** [review]

**Pull request merged**